src/version.cmake

```diff
@@ -0,0 +1,6 @@
+EXECUTE_PROCESS(
+	COMMAND /usr/bin/git describe --tags --always --dirty --match "[0-9]*.[0-9]*"
```

Comment:
Absolute path to git will probably fail on platforms other then Unix-like. Since you did "find git" before maybe just use GIT_EXECUTABLE variable?